services/app-api/src/emailer/emails/unlockContractCMSEmail.ts

```diff
@@ -0,0 +1,80 @@
+import { formatCalendarDate } from '../../../../app-web/src/common-code/dateHelpers'
+import type { UnlockedContractType } from '../../domain-models'
+import { packageName as generatePackageName } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+
+import {
+    stripHTMLFromTemplate,
+    renderTemplate,
+    findContractPrograms,
+    generateCMSReviewerEmailsForContract,
+} from '../templateHelpers'
+import type { EmailData, EmailConfiguration, StateAnalystsEmails } from '../'
+import type { ProgramType, UpdateInfoType } from '../../domain-models'
+
+export const unlockContractCMSEmail = async (
```

Comment:
Could you add tests for CMS and State unlock emailers. You can copy over the relevant ones from the old emailers.

Reply:
Yup that is one of the todos for this PR

Reply:
Whoops! Totally missed that sorry.

Reply:
No worries! I just opened the PR early because I was leaving to be out on Friday